sdk/internal/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Release History
 
-## 1.10.1 (Unreleased)
+## 1.11.0-beta.1 (Unreleased)
```

Comment:
Is this supposed to be 1.10.1? Not sure why we are failing the pipeline :(

Reply:
It's complaining because this file should match the version you have in the changelog: https://github.com/Azure/azure-sdk-for-go/blob/e52a629c15ce39c9d9de7e0bc707853c67bea1c8/sdk/internal/version.go

Just checking though, this is purely internal functionality for testing. Do we need to do a beta for this @jhendrixMSFT?

Reply:
Ah right.. forgot about that one 😅

Reply:
I checked and didn't see any beta versions in the change log, so updated both to 1.11.0